lib/model/testutils_test.go

```diff
@@ -114,6 +115,21 @@ func setupModel(w config.Wrapper) *model {
 	return m
 }
 
+func newModel(cfg config.Wrapper, id protocol.DeviceID, clientName, clientVersion string, ldb *db.Lowlevel, protectedFiles []string) *model {
+	return NewModel(cfg, id, clientName, clientVersion, ldb, protectedFiles).(*model)
+}
+
+func stopModel(m *model) {
+	m.Stop()
+	m.db.Close()
+	os.Remove(m.cfg.ConfigPath())
```

Comment:
This is a slightly surprising action by something called just `stopModel`, but if it's only called from contexts where that makes sense it's OK by me under the circumstances

Reply:
Yeah, it actually just stops and cleans up everything that's used when a model is running during tests. Would `stop`, `cleanupModel` or `cleanupTestModel` reflect that better?

Reply:
cleanupSomething would be better I think yes. But it’s not a big deal, given that we always start the model with a tmpdir config. Do whatever and merge. :)